benches/benches/benchmarks/util.rs

```diff
@@ -60,7 +60,7 @@ pub fn new_always_success_chain(txs_size: usize, chains_num: usize) -> Chains {
 
     let genesis_block = BlockBuilder::default()
         .dao(dao)
-        .difficulty(U256::from(1000u64).pack())
+        .compact_target(0x1f41_8937u32.pack()) // diff 1000
```

Comment:
I think it's better to use a method like `diff_to_compact_target` to avoid the magic number and the comment.